src/Frontend.elm

```diff
@@ -16,24 +16,24 @@ import AssocList as Dict exposing (Dict)
 import AssocSet as Set
 import Browser exposing (UrlRequest(..))
 import Cache exposing (Cache(..))
-import Colors exposing (UserConfig)
 import CreateGroupPage
 import DictExtra as Dict
 import Duration
-import Effect.Browser.Dom as Dom
+import Effect.Browser.Dom as Dom exposing (HtmlId)
 import Effect.Browser.Events as BrowserEvents
 import Effect.Browser.Navigation as BrowserNavigation exposing (Key)
 import Effect.Command as Command exposing (Command, FrontendOnly)
 import Effect.Lamdera
 import Effect.Subscription as Subscription exposing (Subscription)
 import Effect.Task as Task
 import Effect.Time as Time
-import Element exposing (Element)
-import Element.Background
-import Element.Border
-import Element.Font
-import Element.Input
-import Element.Region
+import Element exposing (..)
+import Element.Background as Background
```

Comment:
I'm a bit undecided whether to alias `Element.*` modules. For now though, so that it's consistent with existing code, can you change back all the instances of `import Element.* as *` to just being `import Element.*`?

Reply:
Sure, no problem! As I said in Discord, I was kinda, let's go and if you don't like my alias normalisation, I could just go back to where it was before. So sure :)